incubator/virtualcluster/pkg/controller/virtualcluster/virtualcluster_controller.go

```diff
@@ -369,28 +370,61 @@ func (r *ReconcileVirtualcluster) createVirtualcluster(vc *tenancyv1alpha1.Virtu
 // +kubebuilder:rbac:groups=tenancy.x-k8s.io,resources=virtualclusters/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=tenancy.x-k8s.io,resources=clusterversions,verbs=get;list;watch
 // +kubebuilder:rbac:groups=tenancy.x-k8s.io,resources=clusterversions/status,verbs=get
-func (r *ReconcileVirtualcluster) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r *ReconcileVirtualcluster) Reconcile(request reconcile.Request) (rncilRslt reconcile.Result, err error) {
 	log.Info("reconciling Virtualcluster...")
 	vc := &tenancyv1alpha1.Virtualcluster{}
-	err := r.Get(context.TODO(), request.NamespacedName, vc)
-	if err != nil {
-		return reconcile.Result{}, ctrlutil.IgnoreNotFound(err)
+	err = r.Get(context.TODO(), request.NamespacedName, vc)
+	if ctrlutil.IgnoreNotFound(err) != nil {
```

Comment:
We should empty err if it is NotFound err.

Reply:
set NotFound err to nil